guides/source/form_helpers.md

```diff
@@ -871,7 +871,7 @@ Which will result in a `params` hash that looks like:
 
 All of the form inputs map to the `"person"` hash because we called `fields_for`
 on the `person_form` form builder. By specifying an `:index` option, we mapped
-the address inputs to `person[address][#{address.id}][city]` instead of
+the address inputs to `person[address]["#{address.id}"][city]` instead of
```

Comment:
`person[address][#{address.id}][city]` here is the value of the `name` attribute (see the [rendered output above it](https://github.com/rails/rails/blob/main/guides/source/form_helpers.md?plain=1#L849)).  In other words, `person[address][#{address.id}][city]` is itself a string, so there should be no quotes inside of it.

We could do something else (for example, put quotes around the whole thing), but I feel like we already use this notation  (without quotes) to generally denote interpolation elsewhere in the guides and API docs.